pkg/asset/installconfig/installconfig.go

```diff
@@ -131,11 +132,17 @@ func (a *InstallConfig) Load(f asset.FileFetcher) (found bool, err error) {
 
 	config := &types.InstallConfig{}
 	if err := yaml.UnmarshalStrict(file.Data, config, yaml.DisallowUnknownFields); err != nil {
-		if strings.Contains(err.Error(), "unknown field") {
-			err = errors.Wrapf(err, "failed to parse first occurence of unknown field")
-		}
 		err = errors.Wrapf(err, "failed to unmarshal %s", installConfigFilename)
-		return false, errors.Wrap(err, asset.InstallConfigError)
+		if !strings.Contains(err.Error(), "unknown field") {
+			return false, errors.Wrap(err, asset.InstallConfigError)
+		}
+		err = errors.Wrapf(err, "failed to parse first occurence of unknown field")
+		logrus.Warnf(err.Error())
+		logrus.Warnf("trying unmarshal without unknown fields check which may cause errors")
```

Comment:
1. Log messages--unlike error messages (which makes this hard to remember)--should be capitalized
2. Let's switch this to an Info
3. Maybe reword to something like
```suggestion
		logrus.Info("Attempting to unmarshal while ignoring unknown keys because strict unmarshaling failed")
```